IO.go

```diff
@@ -64,3 +77,43 @@ func (io IO) SetDeltaTime(value float32) {
 func (io IO) SetFontGlobalScale(value float32) {
 	C.iggIoSetFontGlobalScale(io.handle, C.float(value))
 }
+
+// KeyPress sets the KeysDown flag
+func (io IO) KeyPress(key int) {
+	C.iggIoKeyPress(io.handle, C.int(key))
+}
+
+// KeyRelease clear the KeysDown flag
+func (io IO) KeyRelease(key int) {
+	C.iggIoKeyRelease(io.handle, C.int(key))
+}
+
+// KeyMap maps a key into the KeysDown array which represents your "native" keyboard state
+func (io IO) KeyMap(imguiKey int, nativeKey int) {
+	C.iggIoKeyMap(io.handle, C.int(imguiKey), C.int(nativeKey))
+}
+
+// KeyCtrl sets the keyboard modifier control pressed
+func (io IO) KeyCtrl(leftCtrl int, rigthCtrl int) {
+	C.iggIoKeyCtrl(io.handle, C.int(leftCtrl), C.int(rigthCtrl))
+}
+
+// KeyShift sets the keyboard modifier shift pressed
+func (io IO) KeyShift(leftShift int, rigthShift int) {
+	C.iggIoKeyShift(io.handle, C.int(leftShift), C.int(rigthShift))
+}
+
+// KeyAlt sets the keyboard modifier alt pressed
+func (io IO) KeyAlt(leftAlt int, rigthAlt int) {
+	C.iggIoKeyAlt(io.handle, C.int(leftAlt), C.int(rigthAlt))
+}
+
+// KeySuper sets the keyboard modifier super pressed
+func (io IO) KeySuper(leftSuper int, rigthSuper int) {
+	C.iggIoKeySuper(io.handle, C.int(leftSuper), C.int(rigthSuper))
+}
+
+// AddInputCharactersUTF8 adds a new character into InputCharacters[].
+func (io IO) AddInputCharactersUTF8(chars string) {
+	C.iggIoAddInputCharactersUTF8(io.handle, C.CString(chars))
```

Comment:
memory leak: the pointer returned by C.CString needs to be freed!
use the helper function `wrapString() (*C.char, func())` in imgui package to do the necessary tasks. See other cases for example usage, e.g. imgui.Text() .

Reply:
Oeps! I am new to Golang. I use this to learn. And I have. Thanks!